Update systemd generators

zpytela opened the pull request.

martinpitt commented:
@zpytela Please ignore or retry the cockpit failure. That got fixed yesterday in https://github.com/cockpit-project/cockpit/pull/19853 Thanks!

zpytela committed:
Update policy for systemd generators

Update the systemd_generator template.
Update the existing generators policy for the updated template
together with a general clean-up.

zpytela committed:
Add policy for second batch of generators

The following systemd system generators were confined:
- bless-boot
- cryptsetup
- debug
- getty
- zram

zpytela committed:
Add policy for ssh-generator

A small new unit generator "systemd-ssh-generator" has been added to
systemd in v256. It checks if the sshd binary is installed. If so, it
binds it via per-connection socket activation to various sockets
depending on the execution context.

https://github.com/systemd/systemd/releases/tag/v256-rc1

zpytela committed:
Add policy for tpm2 generator

A new unit tpm2.target has been added to systemd in v256
to provide a synchronization point for units which expect the TPM
hardware to be available. A new generator "systemd-tpm2-generator"
has been added that will insert this target whenever it detects that
the firmware has initialized a TPM, but Linux hasn't loaded a driver
for it yet.

https://github.com/systemd/systemd/releases/tag/v256-rc1

zpytela committed:
Add policy for a generic generator

All systemd system generators which do not have a particular type
assigned will now be executed in the systemd_generic_generator_t domain
instead of init_t.
